index.html

```diff
@@ -5,11 +5,9 @@
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
   </head>
-
-  <body>
-
-   
-    <h1 id="title">freeCodeCamp Survey Form</h1>
+<body>
+<section>
+   <h1 id="title">freeCodeCamp Survey Form</h1>
     <p id="description">Thank you for taking the time to help us improve the platform</p>
```

Comment:
- [ ] `inline comment example`
- [ ] Good job using the id :clap: :clap: 
- [ ] however, The basic rule that you need to keep in mind while using classes and ids in CSS is that id is used for single elements that appear on the page for only once (e.g. header, footer, menu), whereas class is used for single or multiple elements that appear on the page for once or more than once (e.g. paragraphs, links, buttons, input boxes). While you can also use a class for a single element, in order to get used to this distinction and use these separators as they are intended, `I suggest` you make a habit of using classes to control the style of multiple of the same type of element.

Reply:
Understood sir

Reply:
Thank you